services/frontend-service/src/ui/Releases.test.tsx

```diff
@@ -72,36 +86,68 @@ describe('Releases', () => {
         expect(container.querySelector('.details')).toBeTruthy();
     });
 
+    it('colors the different releases based on freshness', () => {
+        // given
+        const now = new Date();
+        const hr = 60 * 60 * 1000;
+
+        // when - just created
+        const { container, rerender } = getWrapper({ data: getDummyOverview(now) });
+        // then - new
+        expect(container.querySelector('.releases .details-new')).toBeTruthy();
+
+        // when - one hour old
+        rerender(getNode({ data: getDummyOverview(new Date(now.valueOf() - hr)) }));
+        // then - new
+        expect(container.querySelector('.releases .details-new')).toBeTruthy();
+
+        // when - 12 hours old
+        rerender(getNode({ data: getDummyOverview(new Date(now.valueOf() - 12 * hr)) }));
+        // then - medium
+        expect(container.querySelector('.releases .details-medium')).toBeTruthy();
+
+        // when - 2 days old
+        rerender(getNode({ data: getDummyOverview(new Date(now.valueOf() - 48 * hr)) }));
+        // then - old
+        expect(container.querySelector('.releases .details-old')).toBeTruthy();
+
+        // when - 10 days old
+        rerender(getNode({ data: getDummyOverview(new Date(now.valueOf() - 10 * 24 * hr)) }));
+        // then - history
+        expect(container.querySelector('.releases .details-history')).toBeTruthy();
+
+        // when - not specified
+        rerender(getNode());
+        // then - history
+        expect(container.querySelector('.releases .details-history')).toBeTruthy();
```

Comment:
please transform this into a table test.
Input is something like: `hoursToSubtract`,
expectedOutput: detailsCssClass (or something like that)

Reply:
Done!